themes/developer/developer.theme.sh

```diff
@@ -0,0 +1,229 @@
+#! bash oh-my-bash.module
+
+SCM_THEME_PROMPT_DIRTY=" ${_omb_prompt_brown}✗"
+SCM_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" ${_omb_prompt_green}|"
+SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${_omb_prompt_brown}✗"
+GIT_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${_omb_prompt_green}|"
+GIT_THEME_PROMPT_SUFFIX="${_omb_prompt_green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+function return_delimited {
+	# 1 = flag 2 = tag 3 = value
+	if [[ $1 = "true" ]]; then
+		printf "$2 $3---"
+	elif [[ $1 = "false" ]]; then
+		printf "$2 $3"
+	else
+		printf "$1 $2"
+	fi
+}
+
+function extract_key {
+	value=$(echo "$1" | grep -oP "(?<=$2)[^---]+")
+	printf "$value"
+}
+
+function __bobby_clock {
+	printf "$(clock_prompt) "
+
+	if [ "${THEME_SHOW_CLOCK_CHAR}" == "true" ]; then
+		printf "$(clock_char) "
+	fi
+}
+
+function node_version {
+	val_node=$(node --version)
+	if command -v nvm &>/dev/null; then
+		return_delimited $1 "node" "nvm ${val_node}"
```

Comment:
```suggestion
		return_delimited "$1" "node" "nvm ${val_node}"
```

Can you properly quote the arguments? If you intend to split the arguments, please consider using `_omb_util_split` for a safer splitting. If you intend to perform the glob expansion, please consider using `_omb_util_glob_expand`. (As far as I see the implementation of `return_delimited`, I guess neither word splitting nor glob expansions are expected here.) There are several similar cases in this PR.

Reply:
I now understand why this was not quoted; the first argument of `return_delimited` (`true` or `false`) has been an optional argument. Anyway, this interface was removed in commit b4a9b9d, and all those `$1` do not exist anymore. We can just mark this conversation as resolved.